community/chromium/build

```diff
@@ -62,16 +68,11 @@ mv -f _ tools/generate_shim_headers/generate_shim_headers.py
 # libvpx: Broken ?
 # zlib: KISS build without 'minizip'.
 system="
-    ffmpeg
     flac
     fontconfig
-    freetype
-    harfbuzz-ng
     libdrm
     libjpeg
     libwebp
-    libxml
-    libxslt
```

Comment:
Is there a reason for this horrific development?! :O

Reply:
I mentioned it in the commit message. System freetype harfbuzz is broken and system ffmpeg is older than bundled ffmpeg so if we patched the bundled ffmpeg (https://github.com/archlinux/svntogit-packages/tree/packages/chromium/trunk) we would have to force a rebuild when system ffmpeg updates (Since the changes are ABI breaking). libxml/xslt are just needed by browsers and you usually just have 1 browser on your system so i bundled them